pkgs/by-name/pt/pterm/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, gcc11Stdenv
+, fetchurl
+, libsndfile
+, wxGTK31
```

Comment:
this may be dropped soon from nixpkgs, is it possible to patch to use 3.2?

Reply:
I could have sworn it didnt like 3.2 but its building for me no problem now :shipit: